blockchain/package.json

```diff
@@ -1,9 +1,19 @@
 {
-  "name": "DungeonDefendersContracts",
+  "name": "dungeon-defenders-contracts",
+  "version": "0.1.16",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "private": true,
   "scripts": {
     "compile": "hardhat compile",
     "test": "hardhat test",
-    "deploy": "ts-node --files ./scripts/deploy.ts",
+    "module": "npm run module:build && npm run module:updateDependents",
+    "module:build": "npx tsc",
+    "module:updateDependents": "cd ../frontend && npm update dungeon-defenders-contracts && cd ../server && npm update dungeon-defenders-contracts",
+    "deploy": "npx hardhat run ./scripts/deploy.ts --network goerli",
+    "verify": "npx hardhat run ./scripts/verify.ts --network goerli",
+    "deployAndVerify": "npm run deply && npm run verify",
+    "deployVerifyModule": "npm run deploy && npm run verify && npm run module",
```

Comment:
@ablin42 this will deploy contracts, verify them on etherscan, and update frontend's and server's dependency